src/api.js

```diff
@@ -1,16 +1,50 @@
-import fetch from "unfetch";
+import fetch from "isomorphic-unfetch";
+import https from "https";
+
+const nodeGetPokemons = () => {
+  return new Promise((resolve, reject) => {
+    https
```

Comment:
the https module only works in Node, not in the browser.

Reply:
Actually it's being called only for the node piece and for the browser we still you fetch